### PR TITLE
Change Jetlore JSONP function to use image tags instead of script tags

### PR DESCRIPTION
**Pull request by zacharyhill**
Doing this since we cannot get a CSP exception to hit Jetlore.

### CODE REVIEW

File: src/lib/jetlore.js [hunk 1]
```diff
@@ -311,12 +311,11 @@ const getJetlore = () => {
...
       _jsonp: function(self, url) {
-          var scriptTag = document.createElement("script");
-          scriptTag.type = "text/javascript";
-          scriptTag.src=url;
-          scriptTag.async = true;
-          document.getElementsByTagName("head")[0].appendChild(scriptTag);
-          self.log("SENT JSONP request: " + url)
+        var imgTag = document.createElement("img");
+        imgTag.src=url;
+        imgTag.style.display = 'none'
```
Comment (songz) [inline src/lib/jetlore.js]: in some browsers, display none might cause browser not to load it. Could you instead, style position fixed top left, with width and height of 1
https://stackoverflow.com/questions/12158540/does-displaynone-prevent-an-image-from-loading

File: src/lib/jetlore.js [hunk 2]
```diff
@@ -311,12 +311,11 @@ const getJetlore = () => {
...
+        document.getElementsByTagName("body")[0].appendChild(imgTag);
```
Comment (songz) [inline src/lib/jetlore.js]: y not just document.body? `getElementsByTagName` will do a dom traversal

File: src/lib/jetlore.js [hunk 3]
```diff
@@ -311,12 +311,11 @@ const getJetlore = () => {
...
+        self.log("SENT JSONP request: " + url)
```
Comment (songz) [inline src/lib/jetlore.js]: its not a jsonp request! `sent tracking request`

Comment (zacharyhill) [reply src/lib/jetlore.js]: That was the log they had in there originally.. Do you really want me to change it? The function is called _jsonp. This log goes to them not the browser.

Comment (songz) [reply src/lib/jetlore.js]: no

**Comment by songz** [comment]
closing in preference for this: https://github.com/paypal/paypal-muse-components/pull/21

**Pull request closed without merging**